### PR TITLE
GEODE-2513 Unbrand docs section on Preserving Data

### PR DESCRIPTION
**Pull request by karensmolermiller**
- Remove references to GemFire
- Fix typos
- Clarify links that leave the native-manual

@davebarnes97 @joeymcallister @echobravopapa @mmartell @PivotalSarge @dgkimura Please review this PR containing changes to the user manual.

### CODE REVIEW

File: docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb [hunk 1]
```diff
@@ -19,19 +19,19 @@ See the License for the specific language governing permissions and
...
 
-To assist with tuning, Geode provides statistics that track message queues for durable clients through the disconnect and reconnect cycles. For statistics documentation, see [Statistics](geodeman/managing/statistics/chapter_overview.html).
+To assist with tuning, Geode statistics track message queues for durable clients through the disconnect and reconnect cycles.
 
-When the queue is full, it blocks further operations that add messages until the queue size drops to an acceptable level. The action to take is specified on the server. For details on configuring the queue, see [Implementing Durable Client/Server Messaging](geodeman/developing/events/implementing_durable_client_server_messaging.html).
+When the queue is full, it blocks further operations that add messages until the queue size drops to an acceptable level. Server configuration sets the action to take. See details on server configuration of the queue in the server documentation section [Implementing Durable Client/Server Messaging](geodeman/developing/events/implementing_durable_client_server_messaging.html).
```
Comment (echobravopapa) [inline docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb]: That is much clearer instruction than before

File: docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb [hunk 2]
```diff
@@ -19,19 +19,19 @@ See the License for the specific language governing permissions and
...
 
-## Configuring a Durable Native Client Using gfcpp.properties
+## Configuring a Durable Client Using gfcpp.properties
```
Comment (echobravopapa) [inline docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb]: I think this file is called "geode.properities" now

**Comment by joeymcallister** [comment]
+1 (after @echobravopapa's suggested change of "gfcpp.properties" to "geode.properties," if that file name has changed)

File: docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb (hunk 2 above)
Comment (davebarnes97) [inline docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb]: True. gfcpp.properties was changed to geode.properties with PR #13.

Comment (karensmolermiller) [inline docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb]: I propose that we find all instances of this property file name within the docs and update them all in a separate PR.

Comment (joeymcallister) [inline docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb]: +1

Comment (echobravopapa) [inline docs/geode-native-docs/preserving-data/configuring-durable-nc.html.md.erb]: works for me

**Comment by karensmolermiller** [comment]
Merged in. Closing.

**Pull request closed without merging**